src/box/box.cc

```diff
@@ -1705,11 +1728,25 @@ box_set_replication_connect_timeout(void)
 void
 box_set_replication_connect_quorum(void)
 {
+	if (bootstrap_strategy == BOOTSTRAP_STRATEGY_AUTO) {
+		say_warn("Setting 'replication_connect_quroum' takes no "
+			 "effect with bootstrap_strategy = 'auto'");
+		return;
+	}
 	replication_connect_quorum = box_check_replication_connect_quorum();
 	if (is_box_configured)
 		replicaset_check_quorum();
 }
 
+int
+box_set_bootstrap_strategy(void)
+{
+	enum bootstrap_strategy strategy = box_check_bootstrap_strategy();
+	if (strategy == BOOTSTRAP_STRATEGY_INVALID)
+		return -1;
+	bootstrap_strategy = strategy;
+	return 0;
+}
 void
```

Comment:
Missing an empty line between the functions.

Reply:
Fixed. Thanks!